schemas/org.ietf/http_client_hints/jsonschema/1-0-0

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
```

Comment:
Do we want a description on this schema to tell us what it is for?  There are also no descriptions on any of the fields..